packages/auto-label/src/auto-label.ts

```diff
@@ -354,6 +354,27 @@ export function handler(app: Application) {
     const owner = context.payload.repository.owner.login;
     const repo = context.payload.repository.name;
     const pull_number = context.payload.pull_request.number;
+
+    if (config.product) {
+      const driftRepos = await handler.getDriftRepos();
+      if (!driftRepos) {
+        return;
```

Comment:
if no drift repos were found do we want to skip all subsequent steps? or just the `addLabeltoRepoAndIssue` step?